tonic/src/client/grpc.rs

```diff
@@ -71,8 +71,8 @@ impl<T> Grpc<T> {
     ) -> Result<Response<M2>, Status>
     where
         T: GrpcService<BoxBody>,
-        T::ResponseBody: Body + HttpBody + Send + 'static,
-        <T::ResponseBody as HttpBody>::Error: Into<crate::Error>,
+        T::ResponseBody: Body + Send + Sync + 'static,
```

Comment:
Sync is new here?

Reply:
Yes and no. The old `tonic::body::Body` [had `Sync` as a super trait](https://docs.rs/tonic/0.4.3/src/tonic/body.rs.html#16-42) so previously `Sync` was implicitly required.

Reply:
Make sense!